pubspec.yaml

```diff
@@ -34,9 +34,10 @@ dependencies:
   # Discover network info and configure themselves accordingly
   network_info_plus: ^3.0.1
   # Helps you discover open ports, devices on subnet and more.
-  network_tools: ^3.2.1
-  # network_tools:
-  #   path: ../network_tools/network_tools
+  network_tools: ^3.2.2
```

Comment:
`network_tools_flutter` should include `network_tools` and expose everything that the user needs.
There is no reason users will have to add 2 packages.

Reply:
The imported package 'network_tools' isn't a dependency of the importing package.
Try adding a dependency for 'network_tools' in the 'pubspec.yaml' file.dart[depend_on_referenced_packages](https://dart.dev/lints/depend_on_referenced_packages)

Reply:
So we need to fix it, if we exposed the functions through network_tools_fluttet the programmers will add only one package

Reply:
You can not expose another package I guess. Give me some example otherwise.

Reply:
Create a file like this in network_tools_flutter
https://github.com/osociety/network_tools/blob/dev/lib/network_tools.dart
and point it to the files in network_tools